app/models/stake_pool.rb

```diff
@@ -29,4 +29,13 @@ def average_apy
   def validators_count
     stake_accounts.pluck(:validator_id).compact.uniq.count
   end
+
+  def total_stake
+    stake_accounts&.map(&:delegated_stake).compact.sum
```

Comment:
```suggestion
    stake_accounts&.pluck(:delegated_stake).compact.sum
```
might be a bit faster